Bottom: 61625ad252d9258fe614763c94a672667773801b
Top:    ee4bb1b3b1b5d408ecb373f132744ab041194164
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 15:42:35 +0000

Refresh of play-with-quad-edge-data-STEP

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 0c3cd6b..b781307 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -1,11 +1,12 @@
 #include <stdio.h>
 //#include <stdlib.h>
-//#include <stdbool.h>
+#include <stdbool.h>
 
 #include <glib.h>
 
 #include "hid/common/quad.h"
 #include "hid/common/vertex3d.h"
+#include "hid/common/contour3d.h"
 #include "hid/common/appearance.h"
 #include "hid/common/face3d.h"
 //#include "hid/common/edge3d.h"
